services/device-registry/src/main/java/org/eclipse/hono/deviceregistry/impl/FileBasedCredentialsService.java

```diff
@@ -44,108 +51,82 @@
 @Repository
 @ConfigurationProperties(prefix = "hono.credentials")
```

Comment:
there are no additional properties to be set, are there? You are using the `DeviceRegistryConfigProperties` as indicated by the class parameter ...